Producer/Producer.Shared/Auth/Keychain.cs

```diff
@@ -191,6 +194,20 @@ public bool SaveItemToKeychain (string service, string account, string privateKe
 
 		public bool RemoveItemFromKeychain (string service)
 		{
+			var context = Android.App.Application.Context;
+			var serviceId = $"{context.PackageName}.nomadcode.auth-{service}";
+
+			var item = GetItemFromKeychain (service);
+			var keystore = getKeystore (service);
+			using (var stream = context.OpenFileOutput (serviceId, FileCreationMode.Private))
+			{
+				var tuple = item.ToTuple ();
+				if (!string.IsNullOrWhiteSpace (tuple?.Item1) && !string.IsNullOrWhiteSpace (tuple?.Item1))
```

Comment:
No need to turn this into a Tuple first (item cannot be null):

```c#
if (!string.IsNullOrWhiteSpace (item.Account) && !string.IsNullOrWhiteSpace (item.PrivateKey))
{
    keystore.DeleteEntry (item.Account);
}
```

Reply:
Done, was thinking token key was to be access via item.Token so tried to cast to Tuple Item1.. ima Fool, awesome tuple